docker/dockerfiles/bedrock_l10n

```diff
@@ -1,2 +1,5 @@
 FROM ${FROM_DOCKER_REPOSITORY}:${GIT_COMMIT}
 COPY . /app/locale
+USER root
+RUN chown -R webdev /app/locale
```

Comment:
change to `chown -R webdev.webdev /app/locale` and I think we're good. The current code does change the owner but the group stays `root`.